### PR TITLE
CIV-9407 Trial Arrangements: Email Notification to Citizen on Completion of Trial Arrangements by the Defendant LiP

### PR DESCRIPTION
**Pull request by maciejmodzelewski**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-9407

### Change description ###
Logic driving email notifications send to each party when the other completed their trial arrangements

notify_others_email

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -333,7 +333,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.9'
+  implementation 'com.github.hmcts:civil-commons:1.0.10-civ-9123-civ-9407'
```
Comment (maciejmodzelewski) [inline build.gradle:336]: This is a temporary change for testing purposes. A full release with the required functionality should be added once available.

**Review by LeonardoPalmeiro — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -333,7 +333,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.11'
+  implementation 'com.github.hmcts:civil-commons:1.0.11-civ-9123-civ-9407'
```
Comment (maciejmodzelewski) [inline build.gradle:336]: This is a temporary change for testing purposes. A full release with the required functionality should be added once available.

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by bhagyashreesharma90 — DISMISSED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: build.gradle [hunk 3]
```diff
@@ -333,7 +333,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.12'
+  implementation 'com.github.hmcts:civil-commons:1.0.12-civ-9123-civ-9407'
```
Comment (maciejmodzelewski) [inline build.gradle:336]: This is a temporary change for testing purposes. A full release with the required functionality should be added once available.

**Review by bhagyashreesharma90 — DISMISSED** [review]

File: build.gradle [hunk 4]
```diff
@@ -333,7 +333,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.12'
+  implementation 'com.github.hmcts:civil-commons:1.0.12-civ-9123-9407'
```
Comment (maciejmodzelewski) [inline build.gradle:336]: This is a temporary change for testing purposes. A full release with the required functionality should be added once available.

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by bhagyashreesharma90 — APPROVED** [review]

**Comment by HarryH96** [comment]
Closing PR as ticket is in Blocked

**Comment by HarryH96** [comment]
Removing enable helm and closing PR as ticket is dev blocked

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by bhagyashreesharma90 — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Pull request merged**